SPECS/annobin/annobin.spec

```diff
@@ -1,62 +1,151 @@
-Vendor:         Microsoft Corporation
-Distribution:   Azure Linux
-
 Name:    annobin
 Summary: Binary annotation plugin for GCC
-Version: 9.27
-Release: 4%{?dist}
-License: GPLv3+
-URL:     https://fedoraproject.org/wiki/Toolchain/Watermark
+Version: 12.49
+Release: 1%{?dist}
+License: GPL-3.0-or-later AND LGPL-2.0-or-later AND (GPL-2.0-or-later WITH GCC-exception-2.0) AND (LGPL-2.0-or-later WITH GCC-exception-2.0) AND GFDL-1.3-or-later
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL: https://sourceware.org/annobin/
 # Maintainer: nickc@redhat.com
+# Web Page: https://sourceware.org/annobin/
+# Watermark Protocol: https://fedoraproject.org/wiki/Toolchain/Watermark
 
+#---------------------------------------------------------------------------------
 
-# # Do not build the annobin plugin with annotation enabled.
-# # This is because if we are bootstrapping a new build environment we can have
-# # a new version of gcc installed, but without a new of annobin installed.
-# # (i.e. we are building the new version of annobin to go with the new version
-# # of gcc).  If the *old* annobin plugin is used whilst building this new
-# # version, the old plugin will complain that version of gcc for which it
-# # was built is different from the version of gcc that is now being used, and
-# # then it will abort.
-#
-# Suppress this for BZ 1630550.
-# The problem should now only arise when rebasing to a new major version
-# of gcc, in which case the undefine below can be temporarily reinstated.
+# Set this to zero to disable the requirement for a specific version of gcc.
+# This should only be needed if there is some kind of problem with the version
+# checking logic or when building on RHEL-7 or earlier.
 #
-# %%undefine _annotated_build
-
-# Use "--without tests" to disable the testsuite.  The default is to run them.
-%bcond_without tests
+# Update: now that we have gcc version checking support in redhat-rpm-config
+# there is no longer a great need for a hard gcc version check here.  Not
+# enabling this check greatly simplifies the process of installing a new major
+# version of gcc into the buildroot.
+%global with_hard_gcc_version_requirement 0
 
-# Use "--without annocheck" to disable the installation of the annocheck program.
-%bcond_without annocheck
+#---------------------------------------------------------------------------------
 
-# Use "--with debuginfod" to force support for debuginfod to be compiled into
-# the annocheck program.  By default the configure script will check for
-# availablilty at build time, but this might not match the run time situation.
-%bcond_with debuginfod
+%global annobin_sources annobin-%{version}.tar.xz
+Source: https://nickc.fedorapeople.org/%{annobin_sources}
```

Comment:
Could you move the `Source` tag next to the other ones above? It's hard to find it here.

Reply:
Fixed